modules/math/shared/src/main/scala/gsp/math/Offset.scala

```diff
@@ -49,122 +51,87 @@ object Offset extends OffsetOptics {
     Order.by(o => (o.p, o.q))
 
   /** P component of an angular offset.. */
```

Comment:
It's a trivial detail but maybe we should update the comment (and others below) to not refer to `P` in particular.

Reply:
Good catch. I changed some of the comments but I totally missed a few.